benchmark/redisgraph/generate_graph500_inputs.py

```diff
@@ -1,63 +1,65 @@
+#!/usr/bin/python
```

Comment:
Regarding this entire file and the matching one for the Twitter set:
Thanks to bulk loader improvements, I'm pretty sure all of this logic can be deleted. We just need to prepend each file with a header row, no ID substitution necessary.

That said, this benchmark is too big for me to run locally, so I can't verify that! We don't need to make this change in this PR; there's some value for matching the logic of the previous benchmark run even if it's been obsoleted.

Reply:
agree ( no need to improve an internal benchmark )